opamp/observiq/updater_manager_others.go

```diff
@@ -30,24 +31,39 @@ import (
 // Ensure interface is satisfied
 var _ updaterManager = (*othersUpdaterManager)(nil)
 
+const defaultOthersUpdaterName = "updater"
+
 // othersUpdaterManager handles starting a Updater binary and watching it for failure with a timeout
 type othersUpdaterManager struct {
-	tmpPath string
-	logger  *zap.Logger
+	tmpPath     string
+	cwd         string
```

Comment:
@BinaryFissionGames do we really need to save this? (seems a little odd specifically because it's the current working directory and we only use it once it looks like here)

Reply:
The purpose of the variable is so we can replace it in tests, I'm open to other ideas of injecting the CWD.

Reply:
@BinaryFissionGames ok fair enough, I didn't catch that.